modules/tinymce/src/themes/silver/test/ts/browser/editor/DisableTest.ts

```diff
@@ -0,0 +1,51 @@
+import { ApproxStructure, Assertions, Chain, ChainSequence, Guard, Log, Pipeline, UiFinder } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Editor as McEditor } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+import Theme from 'tinymce/themes/silver/Theme';
+
+UnitTest.asynctest('browser.tinymce.themes.silver.editor.DisableTest', (success, failure) => {
+  Theme();
+
+  const cAssertToolbarState = (label: string, disabled: boolean) => Chain.fromIsolatedChainsWith(SugarBody.body(), [
+    UiFinder.cFindIn('.tox-toolbar'),
+    Chain.control(
+      Assertions.cAssertStructure(label, ApproxStructure.build((s, str, arr) =>
+        s.element('div', {
+          classes: [
+            arr.has('tox-toolbar'),
+            disabled ? arr.has('tox-tbtn--disabled') : arr.not('tox-tbtn--disabled')
+          ],
+          attrs: { 'aria-disabled': str.is(disabled + '') }
+        })
+      )),
+      Guard.tryUntil('Waiting for toolbar state')
+    )
+  ]);
+
+  Pipeline.async({}, [
+    Log.chainsAsStep('TINY-6397', 'Test disable/enable APIs', [
+      McEditor.cFromSettings({
```

Comment:
This looks like it only uses one editor, so this should probably be using `TinyLoader` instead, as `McEditor` is for when we need to init multiple times in the one test.

- [ ] Switch to using `TinyLoader`